cordova_file_opener.ml

```diff
@@ -0,0 +1,116 @@
+[@@@comment "!! This code has been generated by gen_js_api !!"]
```

Comment:
This generated file should not be versioned. Please add an entry in the `.gitignore` file to prevent this from happening.

Reply:
It should be good now (I also rm the ".ml" file from git).